Adding Global Access Agreement (#139217)
elastic/kibana: docs/user/security/access-agreement.asciidoc

agreement before accessing {kib}. The agreement text supports Markdown format and can be specified using the
`xpack.security.authc.providers.<provider-type>.<provider-name>.accessAgreement.message` setting.

> A default access agreement setting can also be specified using `xpack.security.accessAgreement.message`. 

Reviewer: ```suggestion
You can specify a default access agreement  using the `xpack.security.accessAgreement.message` setting. 
```

> This message will be used for each provider that doesn't specify its own access agreement.

Reviewer: ```suggestion
This message will be used for each provider who doesn't specify an access agreement.
```
